.travis.yml

```diff
@@ -22,7 +22,6 @@ branches:
 before_install:
   - pip install -U pip wheel setuptools
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - ./runflake8
```

Comment:
ah sorry, my instruction was not exactly correct, we need to put `flake8` in place of `./runflake8`, otherwise we don't run this check anymore

Reply:
like this here: https://github.com/openwisp/django-freeradius/pull/213
But wait for the build on that PR to complete so we can see if it really works :grin: (it should.. but we never know 100% sure until we try)

Reply:
Oh okay, I was wondering too about ``flake8`` not being ran too... now you cleared my confusion...